packages/ui-components/src/Button/__snapshots__/Button.test.tsx.snap

```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Button: render content can render using children 1`] = `
```

Comment:
Since we are planning enzyme I don't think we should add any snapshots. Shallow rendering won't be an option and we will just remove it anyway.